ci/check_all_ci_scripts_are_run.sh

```diff
@@ -9,7 +9,7 @@ source ci/ci_helpers.sh
 # 2. Strip the directory
 # 3. Exclude some scripts that we should not run in CI directly
 ci_scripts=$(
-    find ci/ -iname "*.sh" |
+    find ci/ -maxdepth 1 -iname "*.sh"  |
```

Comment:
what if we add a folder inside ci folder in the future?

Reply:
See the linked PR. It adds a new folder indeed.